### PR TITLE
aws_ebs_csi_driver: add VolumeSnapshotClass manifest

### PR DESCRIPTION
**Pull request by ipochi**
This PR 

- updates the CRDs to v1
- adds a volumesnapshot class for the aws_ebs_csi_driver
component.

Ideally this  should be a part of the PR #1277.

 ## Release Notes:
 
This change includes addition of Custom resources (CRDs) pertaining to Volume snapshots. The following [CRDs](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd) are added as part of the component update:
 - VolumeSnapshotClasses
 - VolumeSnapshotContents
 - VolumeSnapshots
 
 ### Update Steps:
 ```
 kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/master/assets/charts/components/aws-ebs-csi-driver/crds
 lokoctl component apply aws-ebs-csi-driver
 ```

Signed-off-by: Imran Pochi <imran@kinvolk.io>

### CODE REVIEW

File: assets/charts/components/aws-ebs-csi-driver/templates/volumesnapshotclass.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Lokomotive specific addition.
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-ebs-vsc
```
Comment (invidian) [inline assets/charts/components/aws-ebs-csi-driver/templates/volumesnapshotclass.yaml:5]: `-vsc` here seems redundant.

File: assets/charts/components/aws-ebs-csi-driver/templates/volumesnapshotclass.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Lokomotive specific addition.
+apiVersion: snapshot.storage.k8s.io/v1beta1
```
Comment (surajssd) [inline assets/charts/components/aws-ebs-csi-driver/templates/volumesnapshotclass.yaml:2]: This is `v1` now.

**Review by invidian — CHANGES_REQUESTED** [review]

**Comment by ipochi** [comment]
@surajssd @invidian 
Updated. Have added a label that is required by Velero when using CSI driver.

is this fine or shall I remove the label and add in the CSI PR.

**Review by invidian — DISMISSED** [review]

**Review by knrt10 — DISMISSED** [review]
+1

**Review by surajssd — APPROVED** [review]

**Comment by ipochi** [comment]
@surajssd 

```
installing component "aws-ebs-csi-driver": installing release failed: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "VolumeSnapshotClass" in version "snapshot.storage.k8s.io/v1
```

Fix involves updating the CRDs to `v1`.

**Review by invidian — APPROVED** [review]

**Review by knrt10 — APPROVED** [review]

**Pull request merged**